cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/workers/StatisticsWorker.java

```diff
@@ -121,6 +121,12 @@ private void updateCounters(TimeFrameRecordCache<Counter> counters, Collection<S
 
     @Override
     public void updateEntityBatchStatistics(BookId bookId, BookStatisticsRecordsCaches.EntityKey entityKey, Collection<SerializedEntityMetadata> batchMetadata) {
+        if (interval == 0) {
```

Comment:
it is technically correct, but for more clear solution introduce boolean variable that determines if we are maintaining statistics or not and make decisions according its value